cmd/zed/agents/zfs_mod.c

```diff
@@ -445,16 +500,29 @@ zfs_iter_vdev(zpool_handle_t *zhp, nvlist_t *nvl, void *data)
 	} else if (dp->dd_compare != NULL) {
 		/*
 		 * NOTE: On Linux there is an event for partition, so unlike
-		 * illumos, substring matching is not required to accomodate
+		 * illumos, substring matching is not required to accommodate
```

Comment:
"The maintainers write illumos in lowercase since some computer fonts do not clearly distinguish a lowercase L from an uppercase i (see homoglyph)."

(wikipedia)